config/build/tsconfig.json

```diff
@@ -0,0 +1,51 @@
+{
+  "compilerOptions": {
+    "allowJs": false,
+    "checkJs": false,
+    "alwaysStrict": false,
+    "declaration": false,
+    "declarationMap": false,
+    "downlevelIteration": true,
+    "inlineSources": false,
+    "importHelpers": true,
+    "lib": ["esnext", "dom"],
+    "module": "esnext",
+    "moduleResolution": "node",
+    "noEmitHelpers": true,
+    "noFallthroughCasesInSwitch": true,
+    "noImplicitAny": false,
+    "noImplicitReturns": true,
+    "noImplicitUseStrict": true,
+    "noImplicitThis": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "pretty": false,
+    "resolveJsonModule": true,
+    "sourceMap": true,
+    "strict": true,
+    "target": "es5",
+    "strictBindCallApply": false,
+    "experimentalDecorators": true,
+    // Skip type checking of all declaration files
+    "skipLibCheck": true,
+    "esModuleInterop": true,
+    "jsx": "preserve",
+    "baseUrl": "../../",
```

Comment:
hmm, imo, I'd have it as `config/tsconfig.build.json` to remove a level of nested dirs